app/views/admin/role_appointments/edit.html.erb

```diff
@@ -1,40 +1,74 @@
-<% page_title "Edit appointment to #{@role_appointment.role.name}"  %>
+<% content_for :page_title, "Edit appointment to #{@role_appointment.role.name}"  %>
+<% content_for :title, "Edit appointment to #{@role_appointment.role.name}" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @role_appointment, parent_class: "role_appointment")) %>
```

Comment:
```
<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @role_appointment)) %>
```

This won't break anything, but we only need to pass a `parent_class` in when the model is inheriting from a parent class and the fields use the parent class.

For example, we use one on the edit edition page as the names of the fields are `edition[title]` etc. but the class of the objects if you call`@edition.class` is `Consultation, Case Study` etc. 

Without passing in the parent_class to the error summary it would look for fields names `consultation[title]` etc.

Let me know if that doesn't make sense

Reply:
Cool, yes that makes perfect sense. I can see how the ErrorSummaryComponent handles this now.